LoopCore/NSUserDefaults.swift

```diff
@@ -20,6 +20,8 @@ extension UserDefaults {
         case lastProfileExpirationAlertDate = "com.loopkit.Loop.lastProfileExpirationAlertDate"
         case allowDebugFeatures = "com.loopkit.Loop.allowDebugFeatures"
         case allowSimulators = "com.loopkit.Loop.allowSimulators"
+        case studyProductSelection = "com.loopkit.Loop.studyProductSelection"
```

Comment:
is this a duplicate to what is in `TidepoolKit`? If so, remove from `TidepoolKit`?

Reply:
Removed the duplicate. It is needed in TidepoolKit and not in Loop.